modules/imgcodecs/src/grfmt_png.cpp

```diff
@@ -259,6 +259,19 @@ bool  PngDecoder::readHeader()
             if (!read_from_io(&sig, 8))
                 return false;
 
+            png_uint_32 wdth, hght;
+            int bit_depth, color_type;
+
+            png_read_info( png_ptr, info_ptr );
```

Comment:
When the png_read_info( png_ptr, info_ptr ); function is called, if there are acTL and fcTL chunks, they will be skipped. For the ongoing manual chunk reading, it is necessary to get the file position back. btw i am not sure this change is necessary. old code returns false if there were a problem. if so there is no need to  `png_read_info`